build_docker.sh

```diff
@@ -98,6 +98,11 @@ if [ -n "$STAGING_DIR" ]; then
     GIT_CHECKOUT_COMMAND="git checkout ${GITHUB_DIR}${GITHUB_TAG}"
     echo "${GIT_CHECKOUT_COMMAND}"
     ${GIT_CHECKOUT_COMMAND}
+    # pull just the large runtime resources into the staging area so they are available to the build
+    # when its done as part of the Docker build so the resources can be compiled into the jar as resources
+    GIT_PULL_LARGE_COMMAND="git lfs pull --include src/main/resources/large/"
```

Comment:
Might be good to mention this command in the README for users who want to build GATK without downloading all of the large test data.

Reply:
Done.